updater/spec/dependabot/dependency_change_spec.rb

```diff
@@ -99,11 +99,37 @@
           files: updated_dependency_files,
           dependencies: updated_dependencies,
           credentials: job_credentials,
-          commit_message_options: commit_message_options
+          commit_message_options: commit_message_options,
+          dependency_group: nil
         )
 
       expect(dependency_change.pr_message).to eql("Hello World!")
     end
+
+    context "when a dependency group is assigned" do
+      it "delegates to the Dependabot::PullRequestCreator::MessageBuilder with the group included" do
+        group = Dependabot::DependencyGroup.new(name: "foo", rules: anything)
+
+        dependency_change = described_class.new(
+          job: job,
+          updated_dependencies: updated_dependencies,
+          updated_dependency_files: updated_dependency_files,
+          dependency_group: group
+        )
+
+        expect(Dependabot::PullRequestCreator::MessageBuilder).
+          to receive(:new).with(
+            source: github_source,
+            files: updated_dependency_files,
+            dependencies: updated_dependencies,
+            credentials: job_credentials,
+            commit_message_options: commit_message_options,
+            dependency_group: group
+          )
+
+        expect(dependency_change.pr_message).to eql("Hello World!")
```

Comment:
🔎 When I read this, I see that the assertion is being made against the mock in the [test setup](https://github.com/dependabot/dependabot-core/pull/7137/files#diff-6254ba183349892acb9556d882f751be30988df63e1949923569dcbb3bb2219cR84-R93). Am I reading that right?

If I am reading that correctly, I'm concerned that we'd be getting a false sense of confidence from the test. I think we might be better off just asserting that the instance was created with the correct arguments. What do you think?